fspiop-api/documents/API-Definition_v1.1.1.md

```diff
@@ -2395,6 +2409,24 @@ If the server is unable to find or create a transfer, or another processing erro
 
 **Table 33 -- PUT /transfers/_{ID}_/error data model**
 
+#### 6.7.5.2 PATCH /transfers/_{ID}_/error
+
+Alternative URI: N/A
+
+Logical API service: **Notify Transfer Information Error**
+
+If the server is unable to commit a transfer, or another processing or validation error occurs, during the processing of a **PUT /transfers/**_{ID}_ callbac, the notification **PATCH**
+
+**/transfers/**_{ID}_**/error** is used. The _{ID}_ in the URI should contain the **transferId** (see [Table 30](#table-30)) that was used for the creation of the transfer, or the _{ID}_ that was used in the [**GET /transfers/**_{ID}_](#6731-get-transfersid). See [Table 34](#table-34) for data model.
```

Comment:
The following should be removed as there shouldn't be an error notification for a GET:

> , or the _{ID}_ that was used in the [**GET /transfers/**_{ID}_](#6731-get-transfersid)

Reply:
Yes Henrik, thanks for clarifying. This doesn't apply to the GET calls. Updated the text here